plugins/Capacity.py

```diff
@@ -26,79 +26,90 @@ class Capacity(Plugin):
     def init(self, logger):
         Plugin.init(self, logger)
 
-        self.errors[35001] = self.def_class(
-            item=3500,
+        self.errors[30912] = self.def_class(
+            item=3091,
             level=2,
             tags=["tag"],
             title=T_("Invalid capacity value"),
-            detail=T_("""A capacity tag value is incorrect."""),
+            detail=T_("""Capacity tags should be positive integers."""),
+            resource="https://wiki.openstreetmap.org/wiki/Key:capacity",
+        )
+        self.errors[30913] = self.def_class(
+            item=3091,
+            level=2,
+            tags=["tag"],
+            title=T_("Specific capacity is greater than total capacity"),
+            detail=T_("""A capacity:* value is greater than the total capacity."""),
```

Comment:
```suggestion
            detail=T_("""A capacity:* value should be lower than the total capacity."""),
```

Reply:
Changed in b939fc85dc648b50702dc4d8c2b1007fb39c3310